src/services/expenses.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+let expenses = [];
+
+function emptyExpenses() {
+  expenses = [];
+
+  return expenses;
+}
+
+function getAll({ userId, category, from, to }) {
+  return expenses.length
+    ? expenses.filter(expense => {
+      const isUserIdMatch = userId
+        ? expense.userId === +userId
+        : true;
+
+      const isCategoryMatch = category
+        ? expense.category === category
+        : true;
+
+      const isFromMatch = from
+        ? expense.spentAt >= from
+        : true;
+
+      const isToMatch = to
+        ? expense.spentAt <= to
+        : true;
+
+      return isUserIdMatch && isCategoryMatch && isFromMatch && isToMatch;
+    })
+    : [];
```

Comment:
If u call filter for an empty array it returns []. Remove redundant ternary operator

Reply:
fixed